languages/ruby/exercises/concept/floating-point-numbers/.docs/hints.md

```diff
@@ -0,0 +1,14 @@
+### General
+
+### 1. Calculate the interest rate
+
+- Using an `if` or `case` statement can be useful when checking conditions.
+
+### 2. Calculate the annual balance update
+
+- When calculating the annual yield, it might be useful to temporarily convert a negative balance to a positive one. The `Float` class has a method to convert both positive and negative values to their absolute value.
+
+### 3. Calculate the years before reaching the desired balance
+
+- To calculate the years, one can keep looping until the desired balance is reached.
+- Ruby has no increment operator (`i++`) like some other languages do. Instead, constructs like `i += 1` (which is equal to `i = i + 1`) can be used.
```

Comment:
This line gives away the solution, Maybe rephrase and link to an external resource?

Reply:
You mean the second line about the increment operator, right?

Reply:
Yep!

Reply:
I couldn't find a proper external resource that covers only `+=` without mentioning other things (maybe a few SO answers, but I don't think that's what we're looking for). Giving it a second though, knowing about `+=` isn't crucial to solving the exercise, so maybe it's not even necessary during the exercise. I have moved it to the `after.md` file so the student learns about it but it does not influence the solving of the exercise. Is that ok?